src/NUnitFramework/framework/Api/FrameworkPackageSettings.cs

```diff
@@ -52,7 +52,7 @@ public static class FrameworkPackageSettings
 
         /// <summary>
         /// Full path of the directory to be used for work and result files.
-        /// This path is provided to tests by the frameowrk TestContext.
+        /// This path is provided to tests by the framework TestContext.
```

Comment:
Same here with `<see cref="TestContext"/>`. Gives really nice IntelliSense coloring and help linking.